CHANGELOG.md

```diff
@@ -112,6 +112,9 @@
 - Relax multipart MIME type parsing. A space is allowed in between values
   of the Content-Type header.
   [#2215](https://github.com/Mashape/kong/pull/2215)
+- Multiple auth plugins would overwrite eachothers results, causing 
+  false negatives in an OR scenario.
+  [#2222](https://github.com/Mashape/kong/pull/2222)
```

Comment:
This was not moved to the correct version this was merged against. Be extra wary when merging PRs.